Readme.asciidoc

```diff
@@ -1,68 +1,116 @@
+|APPLICATION_NAME          |Yes          |Application name                                                                         |caching-service-app
+|IMAGE                     |Yes          |Image that will be used to bootstrap the service                                         |jboss-dataservices/datagrid-online-services
+|NUMBER_OF_INSTANCES       |Yes          |Number of instances                                                                      |1
+|TOTAL_CONTAINER_MEM       |No           |Total container size (both limits and requests)                                          |512
+|APPLICATION_USER          |Yes          |Username used for authentication                                                         |<none>
+|APPLICATION_USER_PASSWORD |No           |Password used for authentication                                                         |<generated>
+|======================
 
-The next step is to install the templates. Feel free to use our `Makefile` or manually invoke `oc create -f templates/caching-service.json -n openshift`. After this step you should see the new service in OpenShift UI.
+.Shared Memory Service parameters
+[options="header"]
+|======================
+|Option                    |Is required? |Description                                                                              |Default option
+|NAMESPACE                 |Yes          |Namespace in which this service was created. Used for setting up clustering configuration.|myproject
+|APPLICATION_NAME          |Yes          |Application name                                                                         |caching-service-app
+|IMAGE                     |Yes          |Image that will be used to bootstrap the service                                         |jboss-dataservices/datagrid-online-services
+|NUMBER_OF_INSTANCES       |Yes          |Number of instances                                                                      |1
+|NUMBER_OF_OWNERS          |No           |Number of owners for given piece of data                                                 |2
+|TOTAL_CONTAINER_MEM       |No           |Total container size (both limits and requests)                                          |512
+|APPLICATION_USER          |Yes          |Username used for authentication                                                         |<none>
+|APPLICATION_USER_PASSWORD |No           |Password used for authentication                                                         |<generated>
+|======================
 
-Depending on your OpenShift version and whether the Service Catalog is turned on or off, you need to spin up a new Caching Service instance in your Project.
+=== Creating a client application
 
-Once it's done, just use a Hot Rod client (or REST) to connect to it. Below is an example:
+Once you instantiate one of the services, you might want to connect to it using a Hot Rod (or REST) client.
```

Comment:
"...a Hot Rod or REST client." I don't see why REST should be in parentheses unless it is technically different or something in this context. I guess it's slower/not preferred? Maybe leave it if that is why.

Reply:
In general we prefer using a Hot Rod client. But you are absolutely right here - let's remove parenthesis.